server/tests/integration/test_api_group.py

```diff
@@ -21,153 +21,166 @@
 
 
 class GroupAPITest(APITest, APIBaseTestCase):
-	model = models.Group
```

Comment:
Large diff because I converted tabs to spaces. The tests files are inconsistent regarding tabs vs. spaces.